### PR TITLE
fix(pharos-site): change spacing and style on contributing pages and add accessibility section

### PR DESCRIPTION
**Pull request by Lham42**
**This change:** (check at least one)

- [ ] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**
Fixes spacing on grammar and style page. Adds accessibility section to the contributing documentation page.

### CODE REVIEW

File: packages/pharos-site/src/pages/content-style-guide/grammar-and-style/index.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import BestPractices from '../../../components/statics/BestPractices.tsx';
     }}
   >
     <p>
-      This is our house style; for anything not covered here we follow the
+      This is our house style; for anything not covered here we follow the&nbsp;
```
Comment (daneah) [inline packages/pharos-site/src/pages/content-style-guide/grammar-and-style/index.mdx:13]: I think we want it to be a normal ol' breakable space, unless I'm missing anything?

```suggestion
      This is our house style; for anything not covered here we follow the{' '}
```

File: packages/pharos-site/src/pages/contributing/documentation/index.mdx [hunk 2]
```diff
@@ -23,79 +23,99 @@ import PageSection from '../../../components/statics/PageSection.tsx';
 <PageSection title="Writing documentation">
   <PageSection title="Getting started" subSectionLevel={1}>
     <p>
-      To get started, use our &nbsp;
+      To get started, use our&nbsp;
```
Comment (daneah) [inline packages/pharos-site/src/pages/contributing/documentation/index.mdx:26]:
```suggestion
      To get started, use the{' '}
```

File: packages/pharos-site/src/pages/contributing/documentation/index.mdx [hunk 3]
```diff
@@ -23,79 +23,99 @@ import PageSection from '../../../components/statics/PageSection.tsx';
...
       <PharosLink href="https://docs.google.com/document/d/1GDFM4iQsta8MjErF7hEYkFaNMoQNKzEzpM-RWG8JcwM/edit">
         documentation template
-      </PharosLink>&nbsp; to ensure all needed sections are included and consistent.
+      </PharosLink>&nbsp;to ensure all needed sections are included and consistent.
```
Comment (daneah) [inline packages/pharos-site/src/pages/contributing/documentation/index.mdx:29]:
```suggestion
      </PharosLink>{' '}to ensure all needed sections are included and consistent.
```

File: packages/pharos-site/src/pages/contributing/documentation/index.mdx [hunk 4]
```diff
@@ -23,79 +23,99 @@ import PageSection from '../../../components/statics/PageSection.tsx';
...
     </p>
   </PageSection>
   <PageSection title="Writing guidelines" subSectionLevel={1}>
     <p>
-      Please refer to the
+      Please refer to the&nbsp;
```
Comment (daneah) [inline packages/pharos-site/src/pages/contributing/documentation/index.mdx:34]:
```suggestion
      Please refer to the{' '}
```

File: packages/pharos-site/src/pages/contributing/documentation/index.mdx [hunk 5]
```diff
@@ -23,79 +23,99 @@ import PageSection from '../../../components/statics/PageSection.tsx';
...
       <PharosLink href="https://pharos.jstor.org/content-style-guide/voice-and-tone">
         JSTOR content style guide
-      </PharosLink>&nbsp; to gain a better understanding of our collective writing style and structure.
+      </PharosLink>&nbsp;to gain a better understanding of our collective writing style and structure.
```
Comment (daneah) [inline packages/pharos-site/src/pages/contributing/documentation/index.mdx:37]:
```suggestion
      </PharosLink>{' '}to gain a better understanding of our collective writing style and structure.
```

**Review by Niznikr — APPROVED** [review]

**Review by daneah — APPROVED** [review]

**Pull request merged**